serve/routes.js

```diff
@@ -45,7 +45,11 @@ module.exports = function(app, passport){
 
 	// Receives an ajax get request from the client site to create a folder
 	// The request will contain the path where to create folder and usernam
-	app.get('/mkdir', function(req, res) {
+	app.get('/mkdir',data,function(req, res,data) {
```

Comment:
Removing this `data` after '/mkdir'/ works for me. I can see "Error: printed on my terminal where I am running node.

Reply:
what are other changes that you made ?